### ISSUE TITLE
Historical US House Prices

### ISSUE THREAD

**Issue by rufuspollock** [datasets/awesome-data#61]
Available from Shiller's data page: http://www.econ.yale.edu/~shiller/data.htm

Data file is xls: http://www.econ.yale.edu/~shiller/data/Fig2-1.xls

Note we already have case-shiller index so perhaps this is not a core dataset (but nice to have)

**Comment by andreaswpv** [comment]
I'll be taking a look at this one! How do you define which values to integrate? Everything in the source, then just clean and package, or is there a process to define which fields to have?

**Comment by rgrp** [comment]
@andreaswpv i'd go for everything in the source by default - you can use your judgment here and check back in if queries.

**Comment by rgrp** [comment]
@andreaswpv how are you doing?

**Comment by andreaswpv** [comment]
Right. Package is ready, but I think the data is copyright protected, so I cannot publish it.

**Comment by rgrp** [comment]
@andreaswpv in general this kind of data will likely not get legal protection. I think we should go ahead and publish, if necessary explaining in the README where you think there are concerns.

**Comment by rgrp** [comment]
@andreaswpv can you please share somewhere and we can take care of publishing as a core dataset if you are not happy there.

**Comment by uwaheed88** [comment]
@rufuspollock I working on this issue
